protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -35,12 +35,24 @@ func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
 		"k8s-app": c.PodName,
 	}
 
+	// TODO another hardcoded version
+	image := "/etcd:2.2.1"
+	imageRegistry := "gcr.io/google_containers"
+
+	// Test to determine if the container registry has been passed in as a flag.
+	// If so use the provider registry location.
+	if c.ImageSource == "" {
```

Comment:
Instead of passing an image source can we just pass the whole image? That way it can just be 
```
if c.Image == "" {
    c.Image  = defaultImage
}
```